client/public/clientJS/partnerCarousel.js

```diff
@@ -0,0 +1,67 @@
+$(document).ready(function () {
+  'use strict';
+  var smallParent3 = $('.small3'),
+    smallUlLi3 = $('.small3 .small-ul li'),
```

Comment:
please don't use multi declaration

Reply:
Also it might be better to name the variables after what the html portion is doing instead of class names. From this piece of code no one aside from you would know what this is for without looking further